src/include/dfuse_ioctl.h

```diff
@@ -60,6 +59,15 @@ struct dfuse_user_reply {
 	gid_t gid;
 };
 
+struct dfuse_mem_query {
+	uint64_t inode_count;
+	uint64_t fh_count;
+	uint64_t pool_count;
+	uint64_t container_count;
+	ino_t    ino;
+	bool     found;
+};
```

Comment:
Should we future-proof ourselves by adding a `version` to this struct? Just thinking about what would happen if middlware / third-party compiled with this version of the struct, then recompiled a new version of DAOS without recompiling middleware. @mchaarawi I remember we ran into something similar with DFS before. Thoughts?

Reply:
It would be possible to but I think we'd just switch to a new cmd number if we did that, ioctls are integer commands so don't have the same issues as c functions which are resolved by name.